Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -1088,26 +1088,26 @@
-    spread: 30
+    count: 6 #60 heat damage if you hit all your shots, but wide spread
+    spread: 25
 
 - type: entity
-  name: narrow laser barrage
-  id: BulletLaserSpreadNarrow
-  parent: BulletLaser
+  name: laser slug
+  id: BulletLaserSpreadSlug
+  parent: BulletLaserHeavy
   categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)